### ISSUE TITLE
Android-Kotlin : 07.4 Constraint Layout

### ISSUE THREAD

**Issue by shubhmittal07** [girlscript/winter-of-contributing#2292]
### Description

I wanted to contribute into this topic. Can you please assign it to me

### Domain

Android Dev (Kotlin)

### Type of Contribution

Audio, Video, Documentation

### Code of Conduct

- [X] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.

**Comment by shubhmittal07** [comment]
/assign
Documentation

**Comment by shubhmittal07** [comment]
I will be contributing in Documentation part.

**Comment by shubhmittal07** [comment]
Hey, do I have to update the folder structure because it is a new topic and number 7 folder does not exist. ? Sorry for so many doubts @rahulsain

**Comment by rahulsain** [comment]
Yes, you have to make a folder of 07. Topic name, then sub topic readme
file in it

**Comment by shubhmittal07** [comment]
I have just made a PR  @rahulsain . It is my first time contributing, hope everything is fine.
